specification/docs/domains/index.md

```diff
@@ -0,0 +1,11 @@
+---
+        description: List of supported domains in DevTools Protocol
+        title: Domains 
+     ---
+# DevTools Protocol Domains
+
+- [Debugger](debugger.md) - Debugger domain exposes JavaScript debugging capabilities. It allows setting and removing breakpoints, stepping through execution, exploring stack traces, etc.
+
+
+- [Runtime](runtime.md) - Runtime domain exposes JavaScript runtime by means of remote evaluation and mirror objects. Evaluation results are returned as mirror object that expose object type, string representation and unique identifier that can be used for further object reference. Original objects are maintained in memory unless they are either explicitly released.
```

Comment:
I think "either explicitly released" is missing an "or ..." to finish the sentence. I see the source of this is the JSON file, but still commenting here since that JSON file is too big to show inline :)